flocx_market/db/sqlalchemy/contract_api.py

```diff
@@ -0,0 +1,48 @@
+from flocx_market.db.orm import orm
+from flocx_market.db.sqlalchemy.offer_api import OfferApi
+from flocx_market.db.sqlalchemy.contract_model import ContractModel
+from uuid import uuid4
+
+
+class ContractApi(ContractModel):
+    def __init__(self, time_created, bid_id, status, start_time, end_time, cost, offers):
+        self.contract_id = str(uuid4())
+        self.time_created = time_created
+        self.bid_id = bid_id
+        self.status = status
+        self.start_time = start_time
+        self.end_time = end_time
+        self.cost = cost
+
+    def as_dict(self):
+        d = {}
+        for col in self.__table__.columns:
+            if col.name in ['time_created', 'start_time', 'end_time']:
+                d[col.name] = getattr(self, col.name).isoformat()
+            else:
+                d[col.name] = getattr(self, col.name)
+        offer_model_objects = [offer for offer in self.offers.all()]
```

Comment:
For performance reasons, I don't think we want to expand offer ids into offer objects here (or at least, not by default). It is typical for REST APIs to either (a) not expand referenced objects by default (leaving it up to the client) or (b) only do so when explicitly requested.